src/olympia/editors/models.py

```diff
@@ -243,13 +243,67 @@ def base_query(self):
         return q
 
 
-class ViewAllList(ViewQueue):
+class ViewAllList(RawSQLModel):
+    id = models.IntegerField()
+    addon_name = models.CharField(max_length=255)
+    addon_slug = models.CharField(max_length=30)
     guid = models.CharField(max_length=255)
     version_date = models.DateTimeField()
     _author_ids = models.CharField(max_length=255)
     _author_usernames = models.CharField()
     review_date = models.DateField()
     review_version_num = models.CharField(max_length=255)
+    addon_status = models.IntegerField()
+    latest_version = models.CharField(max_length=255)
+    admin_review = models.BooleanField()
+
+    listed = True  # ViewAll for listed or unlisted addons.
+
+    def base_query(self):
+        return {
+            'select': SortedDict([
+                ('id', 'addons.id'),
+                ('addon_name', 'tr.localized_string'),
+                ('addon_status', 'addons.status'),
+                ('addon_slug', 'addons.slug'),
+                ('latest_version', 'versions.version'),
+                ('guid', 'addons.guid'),
+                ('_author_ids', 'GROUP_CONCAT(authors.user_id)'),
+                ('_author_usernames', 'GROUP_CONCAT(users.username)'),
+                ('admin_review', 'addons.adminreview'),
+                ('version_date', 'versions.nomination'),
+                ('review_date', 'reviewed_versions.reviewed'),
+                ('review_version_num', 'reviewed_versions.version'),
+            ]),
+            'from': [
+                'addons',
+                'LEFT JOIN versions ON (versions.id = addons.latest_version)',
+                #  Translations
```

Comment:
given that the line below this comment says `JOIN translations` I feel like this is obvious information?